src/app/messaging/component/message-window/message-window.component.ts

```diff
@@ -55,6 +60,13 @@ export class MessageWindowComponent implements OnInit, OnDestroy {
           (messages => this.data = messages);
   }
 
+  sendMessage(){
```

Comment:
Please format your code (ideally with Prettier) before committing and pushing

Reply:
done